devices.js

```diff
@@ -8904,6 +8904,159 @@ const devices = [
+        configure: async (device, coordinatorEndpoint) => {
+            await bind(device.getEndpoint(1), coordinatorEndpoint, ['genOnOff']);
+            await bind(device.getEndpoint(2), coordinatorEndpoint, ['genOnOff']);
+        },
+    },
+    {
+        zigbeeModel: ['095db3379e414477ba6c2f7e0c6aa026'],
+        model: 'T21W1Z',
+        vendor: 'ORVIBO',
+        description: 'Smart light switch - 1 gang',
+        supports: 'on/off',
+        fromZigbee: [fz.on_off],
+        toZigbee: [tz.on_off],
+    },
+    {
+        zigbeeModel: ['093199ff04984948b4c78167c8e7f47e'],
+        model: 'W40CZ',
+        vendor: 'Orvibo',
+        description: 'Orvibo smart curtain motor ',
+        supports: 'open, close, stop, position',
+        fromZigbee: [fz.ZNCLDJ11LM_ZNCLDJ12LM_curtain_analog_output, fz.cover_position_tilt, fz.ignore_basic_report],
+        toZigbee: [tz.cover_state, tz.cover_position_tilt],
+        onEvent: async (type, data, device) => {
+            if (data.type === 'attributeReport' && data.cluster === 'genAnalogOutput') {
+                await device.endpoints[0].read('genAnalogOutput', ['presentValue']);
+            }
+        },
+    },
+    {
+        zigbeeModel: ['e0fc98cc88df4857847dc4ae73d80b9e'],
+        model: 'R11W2Z',
+        vendor: 'ORVIBO',
+        description: 'In wall switch - 2 gang',
+        supports: 'on/off',
+        fromZigbee: [fz.on_off],
+        toZigbee: [tz.on_off],
+        endpoint: (device) => {
+            return {'l1': 1, 'l2': 2};
+        },
+        meta: {configureKey: 1},
+        configure: async (device, coordinatorEndpoint) => {
+            await bind(device.getEndpoint(1), coordinatorEndpoint, ['genOnOff']);
+            await bind(device.getEndpoint(2), coordinatorEndpoint, ['genOnOff']);
+        },
+    },
+    {
+        zigbeeModel: ['9ea4d5d8778d4f7089ac06a3969e784b'],
+        model: 'R20W2Z',
+        vendor: 'ORVIBO',
+        description: 'In wall switch single Live',
+        supports: 'on/off',
+        fromZigbee: [fz.on_off],
+        toZigbee: [tz.on_off],
+        meta: {configureKey: 1, multiEndpoint: true},
+        configure: async (device, coordinatorEndpoint) => {
+            await bind(device.getEndpoint(1), coordinatorEndpoint, ['genOnOff']);
+            await bind(device.getEndpoint(2), coordinatorEndpoint, ['genOnOff']);
+        },
+        endpoint: (device) => {
+            return {'left': 1, 'right': 2};
+        },
+    },
+    {
+        zigbeeModel: ['da2edf1ded0d44e1815d06f45ce02029'],
+        model: 'SW21',
+        vendor: 'ORVIBO',
+        description: 'Water leakage sensor',
+        supports: 'water leak',
+        fromZigbee: [fz.ias_water_leak_alarm_1],
+        toZigbee: [],
+    },
+    {
+        zigbeeModel: ['72bd56c539ca4c7fba73a9be0ae0d19f'],
+        model: 'SE21',
+        vendor: 'ORVIBO',
+        description: 'Smart emergency button',
+        supports: 'action',
+        fromZigbee: [fz.SE21_action],
```

Comment:
I don't have access to your repo, please add the following converter to `fromZigbee.js`:

```js
    SE21_action: {
        cluster: 'ssIasZone',
        type: 'commandStatusChangeNotification',
        convert: (model, msg, publish, options, meta) => {
            const buttonStates = {
                0: 'off',
                1: 'single',
                2: 'double',
                3: 'hold',
            };

            return {action: buttonStates[msg.data.zonestatus]};
        },
    },
```

Reply:
Ok